app/api/talk.coffee

```diff
@@ -0,0 +1,19 @@
+JSONAPIClient = {Resource} = require 'json-api-client'
+apiClient = require './client'
+authClient = require './auth'
+
+# Staging:
+# https://talk-staging.zooniverse.org
+
+# Local:
+# http://127.0.0.1:3000
+
+talkClient = new JSONAPIClient 'https://talk-staging.zooniverse.org',
```

Comment:
Can we define the possible Talk API hosts in api/config.coffee so it respects any set environment?